src/DynamoUtilities/DynamoFeatureFlagsManager.cs

```diff
@@ -75,6 +75,8 @@ internal void CacheAllFlags()
                     FlagsRetrieved?.Invoke();
 
                 }, null);
+                var formattedFlags = JsonConvert.SerializeObject(AllFlagsCache, Formatting.Indented);
+                OnLogMessage($"retrieved feature flags with value: {formattedFlags}");
```

Comment:
is this just a debugging thing ? or it makes sense for users too?

Reply:
Logging from ff had performance issues in the past. Might OnLogMessage trigger Wpf updates ?

Reply:
It definitely has WPF updates, but this is a single log, and as we get more and more flags, the other log that we have that spits out the std.out is not readable. This makes it much easier IMO for devs to see what flags are enabled when a user gives us their log.